### PR TITLE
Regex and other fixes

### PR DESCRIPTION
**Pull request by Domarkaite**
Added regex and some other fixes

### CODE REVIEW

File: NOTEA/Controllers/UserController.cs [hunk 1]
```diff
@@ -59,7 +64,8 @@ public IActionResult LogIn()
         [HttpPost]
         public IActionResult LogIn(string username, string password)
         {
-            UserModel user = new UserModel(username, password);
+            UserModel user = new UserModel(username, password, "");
+            //PAVOJUS KAS CIA REIKIA EMAILO ALIO NORIU KAD NEREIKETU
```
Comment (AugustinasJarockis) [inline NOTEA/Controllers/UserController.cs:68]: Čia reiks perdaryti šitai

Comment (Domarkaite) [reply NOTEA/Controllers/UserController.cs:68]: taip pritariu

File: NOTEA/Services/LogServices/LogsService.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@ public void SaveExceptionInfo(ExceptionModel exception)
         {
             using (StreamWriter writer = new StreamWriter("Logs//logs.txt", append: true))
             {
-                string info = "Message: " + exception.Message + " Stack trace: " + exception.StackTrace + " Source: " + exception.Source;
+                string info = "Message: " + exception.Message + " Time: " + exception.DateTime + " Stack trace: " + exception.StackTrace + " Source: " + exception.Source;
```
Comment (AugustinasJarockis) [inline NOTEA/Services/LogServices/LogsService.cs:11]: Pls add whitespace after error

Comment (Domarkaite) [reply NOTEA/Services/LogServices/LogsService.cs:11]: ka turi omeny? tuscia eilute ar tiesiog whitespace or

File: NOTEA/Controllers/UserController.cs (hunk 1 above)
Comment (Domarkaite) [reply NOTEA/Controllers/UserController.cs:68]: ten dar kazkur yra panasi vieta bet i'll fix it xd

**Review by AugustinasJarockis — APPROVED** [review]
Fine. Now I need to provide my email

**Pull request merged**